homeassistant/components/thethingsnetwork/entry_settings.py

```diff
@@ -0,0 +1,72 @@
+"""Wrapper for global settings stored in the The Things network entry."""
```

Comment:
I don't think this field is needed since we don't have an options flow at this moment